src/plugins/discover/public/application/apps/main/components/doc_table/components/pager/tool_bar_pagination.tsx

```diff
@@ -27,6 +28,11 @@ interface ToolBarPaginationProps {
   onPageSizeChange: (size: number) => void;
 }
 
+const TOOL_BAR_PAGINATION_STYLES = {
```

Comment:
why not a class?

Reply:
It will be overwritten by EUI classes, we have to apply more specific selector or use inline styles.